### DISCUSSION TITLE
feat(RadioList): RadioList组件是否应该提供form-check级别的样式设置？

### DISCUSSION THREAD

**Discussion by shakugans** [dotnetcore/BootstrapBlazor#4366] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

form-check默认的宽度在PC上看是比较宽的，使用的话会希望更紧凑一点，比如我这边是6个字符的话，会希望设置为80px。
但组件似乎没有把form-check级别的style或者template暴露出来；目前是在<style>中处理，但这样的处理方案似乎不是很好。
一来其实无法精细定义到每个子元素，二来这样出来似乎会污染样式。

### Describe the solution you'd like

把form-check级别的style或者template暴露出来

### Additional context

<style>
    .radio-list .form-check {
        width: 80px; 
    }
</style>
<RadioList Items="@requestItems" @bind-Value=@requestItem style="max-width:190px;min-width190px;" />

**Comment by bb-auto** [comment]
@shakugans Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@shakugans 组件提供了样式变量 

```
--bb-radio-item-width: 200px;
```

更改即可，非常方便
